be/src/vec/exec/format/orc/vorc_reader.cpp

```diff
@@ -1750,6 +1750,11 @@ Status OrcReader::get_next_block(Block* block, size_t* read_rows, bool* eof) {
                 RETURN_IF_CATCH_EXCEPTION(Block::filter_block_internal(block, columns_to_filter,
                                                                        (*_delete_rows_filter_ptr)));
             }
+            // TODO: Check compatibility of deletion vector and lazy reading
+            IColumn::Filter result_filter(block->rows(), 1);
+            _execute_filter_position_delete_rowids(result_filter);
```

Comment:
Is there a performance issue with a regular hive table？

Reply:
These changes also happened in https://github.com/apache/doris/pull/34814. I will rebase when this pr merged.